packages/components/organisms/f-cookie-banner/f-cookie-banner-static/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "description": "Fozzie Cookie Banner configuration to compile via vue cli prerender to vanilla js/css",
+  "license": "Apache-2.0",
+  "scripts": {
+    "build": "yarn install && yarn cache clean --all && yarn exec gulp"
+  },
+  "browserslist": [
+    "extends @justeat/browserslist-config-fozzie"
+  ],
+  "dependencies": {
+    "@justeat/f-cookie-banner": "file:../dist"
+  },
+  "peerDependencies": {
+    "@justeat/browserslist-config-fozzie": ">=1.1.1",
+    "js-cookie": "2.2.1"
+  },
+  "devDependencies": {
+    "@vue/cli-plugin-babel": "~4.5.0",
+    "@vue/cli-plugin-eslint": "~4.5.0",
+    "@vue/cli-service": "~4.5.0",
+    "del": "^4.1.1",
+    "fancy-log": "^1.3.3",
+    "gulp": "^4.0.2",
+    "gulp-concat": "^2.6.1",
+    "gulp-rename": "^2.0.0",
+    "gulp-replace": "^1.1.3",
+    "prerender-spa-plugin": "^3.2.1",
+    "vue-cli-plugin-prerender-spa": "~1.1.6",
+    "vue-template-compiler": "2.6.10"
+  },
+  "eslintConfig": {
+    "root": true,
+    "env": {
+      "node": true
+    },
+    "extends": [
```

Comment:
Can you use the `@justeat/eslint-config-fozzie` package for lining config? I'm not sure if it should just pick this up through traversing up the directory anyway and inheriting it from the `eslint.rc` at the root of the mono-repo?